### DISCUSSION TITLE
Is there a way to know exactly which column changed during an update ?

### DISCUSSION THREAD

**Discussion by Skwall** [Azure/azure-functions-sql-extension#946] [Q&A] [answered]
Hey everyone,

I'd like to know if there is a way using sql trigger to know which column exactly changed during an update for example ?

If the azure function is triggered by an Update operation that only updates the column "City" of a row in my "Address" table,
can I get the name of this particular property ?

Thanks 🙂

**Comment by Charles-Gagnon** [comment, marked as answer]
Unfortunately not at this time, but we have this issue tracking requesting adding that information and will be looking at that as a future enhancement! 

https://github.com/Azure/azure-functions-sql-extension/issues/710

**Comment by Astral100** [comment]
You may want to look into SQL Change tracking. 
It allows you to get the old version of the row and by comparing the 2 rows you can figure out which fields were changed.
Lots of coding, but it can be done, as I had to implement the same.
